main/keyword.c

```diff
@@ -256,3 +256,14 @@ extern void dumpKeywordTable (FILE *fp)
 		}
 	}
 }
+
+extern void addKeywordGroup (struct keywordGroupDefinition *groupdef,
+							 langType language, bool checkDuplication)
```

Comment:
Instead of making `checkDuplicatoin` as a parameter,  I should it a member of keywordGroupDefinition.